zokrates_book/src/zokrates_js.md

```diff
@@ -60,3 +42,98 @@ initialize().then((zokratesProvider) => {
     const verifier = zokratesProvider.exportSolidityVerifier(keypair.vk, "v1");
 });
 ```
+
+## API
+
+##### initialize()
+Dynamically loads binding wasm module containing Rust-generated wasm functions and returns a promise with a ZoKrates provider. 
```

Comment:
This is still a bit obscure, I'm not sure that much detail is required. How about just "returns an initialized ZokratesProvider as a promise" or something?

Reply:
ah right, nice suggestion, no reason to complicate this really